atlas/portal/pkg/svc/zserver.go

```diff
@@ -0,0 +1,219 @@
+package svc
+
+import (
+	models "atlas/portal/internal"
+	"atlas/portal/pkg/pb"
+	responderpb "atlas/responder/pkg/pb"
+	"context"
+	"errors"
+	"github.com/golang/protobuf/ptypes/empty"
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+	"time"
+)
+
+const (
+	// version is the current version of the service
+	version            = "0.0.1"
```

Comment:
Please use git hash version instead of const version.

Reply:
Done. I just delete get version endpoint